src/core/src/op/util/avg_pool_base.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/util/avg_pool_base.hpp"
+
+#include "itt.hpp"
+
+ov::op::util::AvgPoolBase::AvgPoolBase(const Output<Node>& arg,
+                                       const Strides& strides,
+                                       const Shape& pads_begin,
+                                       const Shape& pads_end,
+                                       const Shape& kernel,
+                                       bool exclude_pad,
+                                       RoundingType rounding_type,
+                                       PadType auto_pad)
+    : Op({arg}),
+      m_kernel(kernel),
+      m_strides(strides),
+      m_pads_begin(pads_begin),
+      m_pads_end(pads_end),
+      m_exclude_pad(exclude_pad),
+      m_auto_pad(auto_pad),
+      m_rounding_type(rounding_type) {
```

Comment:
Prefer `{}` initialization.

Reply:
Done in the https://github.com/openvinotoolkit/openvino/pull/23483/commits/0fa32444523ffe467e6cdc7d1321ad0e35c32d08. Thank you